account_invoice_overdue_reminder/models/overdue_reminder_action.py

```diff
@@ -46,6 +50,7 @@ def _reminder_type_selection(self):
             ('mail', _('E-mail')),
             ('phone', _('Phone')),
             ('post', _('Letter')),
+            ('other', _('Other')),
```

Comment:
Other seems to generic for me. Something like 'manual' looks better. Don't you think ? 

same for the function, so validate_other --> validate_manual.